### DISCUSSION TITLE
Longer chat messages

### DISCUSSION THREAD

**Discussion by DanielWillett** [SmartlyDressedGames/Unturned-3.x-Community#3753] [Ideas]
Please allow plugins to send chat messages longer than 2 lines, it looks a bit weird when you have to split messages into multiple, and automatically splitting them gets tricky with rich text.
The netcode can support up to 2047 bytes but the chat will only display 2 lines.

**Comment by Jdance-Media** [comment]
Agreed, would also be nice to see support for longer messages in general, even for players. No idea about the changes required for that though.

**Reply by GazziFX** [reply]
You can check answer here https://github.com/SmartlyDressedGames/Unturned-3.x-Community/issues/3277#issuecomment-1166667875
